frontend/src/styles.css

```diff
@@ -1,2 +1,265 @@
 /* You can add global styles to this file, and also import other style files */
-@import "primeng/resources/primeng.css";
\ No newline at end of file
+@import "primeng/resources/primeng.css";
+@import 'primeicons/primeicons.css';
+
+    :host {
```

Comment:
We should always avoid putting CSS in here whenever possible. If we overwrite a style here, then chances are high that we might be messing with things that PrimeNG or our own components rely on and then we can inadvertently break thing. 

We should move these styles into the proper component.css files or remove them as needed.
The variable declarations for colors are ok to stay, but I'm pretty sure those aren't being used anywhere.

Also, could we add: 
```
html, body  {
    margin: none;
    padding: none;
}
```


I know this might not seem like a big deal, but in web development with frameworks like Angular/React this is a really really big no no.